174 Honeycluster with clickhouse, ETL processing and fixing column types

mebegu opened the pull request.

Commit:
Update db init to create tables, and soc to sink tetragon and kubescape, and convert timestamp to nanoseconds

Commit:
fix indent

Commit:
fix the time conversion

Commit:
make func infalliable

Commit:
change endpoint url, and add port forward

Commit:
fix the endpoint

Commit:
update init json to nanoseconds for timestamp

Commit:
set correct endpoint port

entlein committed:
porting scripts to Make

Signed-off-by: entlein <einentlein@gmail.com>

entlein commented on analysis/database/init.sh:
unless there is anything in those scripts that the Makefiles dont do, please do not use those

entlein committed:
feature: adding helm values for clickhouse

Signed-off-by: entlein <einentlein@gmail.com>

entlein committed:
feature: adding helm values for clickhouse

Signed-off-by: entlein <einentlein@gmail.com>

mebegu committed:
Add ClickHouse table initialization SQL for ETL pipelines (kubescape, tetragon, http, dns, STIX) And fix ETL problems

* Add ClickHouse table initialization SQL for ETL pipelines (kubescape, tetragon, http, dns, STIX)

Implement Makefile target to auto-create ClickHouse tables via temporary Kubernetes pod

Configure Pixie ETL and STIX ETL auto-start in Flask on boot

Add Flask endpoints to control Pixie ETL lifecycle (start/stop)

Set up local ClickHouse port-forwarding workflow for development

Validate ClickHouse connectivity with secure credentials

Clean up ETL controllers and align with modular Flask structure

Fix typing errors with ETLs

---------

Co-authored-by: laborant <laborant@labs.iximiuz.com>

Commit:
delete obsolete scripts

mebegu committed:
fix typo Update values.yaml

mebegu committed:
175 pixie etl trigger parameters and filtering data (#187)

* add request parameters and filters for pixie etl

* fix the filters

---------

Co-authored-by: laborant <laborant@labs.iximiuz.com>

mebegu committed:
175 pixie etl trigger parameters and filtering data (#188)

* add request parameters and filters for pixie etl

* fix the filters

---------

Co-authored-by: laborant <laborant@labs.iximiuz.com>

mebegu committed:
175 pixie etl trigger parameters and filtering data (#189)

* add request parameters and filters for pixie etl

* fix the filters

* finalize test implementation

---------

Co-authored-by: laborant <laborant@labs.iximiuz.com>
Co-authored-by: Mehmet Berk Gürçay <mehmet.gurcay@danfoss.com>

Commit:
Update the readme after etl implementation

Commit:
delete unused, obsolete scripts
